plugin/providers/keboola/resource_keboola_snowflake_writer_tables.go

```diff
@@ -121,6 +143,14 @@ func resourceKeboolaSnowflakeWriterTablesCreate(d *schema.ResourceData, meta int
 			Destination: fmt.Sprintf("%s.csv", mappedTable.TableID),
 		}
 
+		if val, ok:= config["changed_since"]; ok { storageTable.ChangedSince = val.(string)}
+		if val, ok:= config["where_column"]; ok { storageTable.WhereColumn = val.(string)}
+		if val, ok:= config["where_operator"]; ok { storageTable.WhereOperator = val.(string)}
+
+		if q := config["where_values"]; q != nil {
+			storageTable.WhereValues = AsStringArray(q.([]interface{}))
+		}
+
```

Comment:
What happens if the `ChangedSince`, `WhereColumn`, `WhereOperator` and `WhereValues` are omitted/zero-valued during the Create/Update? Any weird results when you apply it?

Reply:
If you look at the struct definition:
```
type SnowflakeWriterStorageTable struct {
	Source          string   `json:"source"`
	Destination     string   `json:"destination"`
	Columns         []string `json:"columns"`
	ChangedSince    string   `json:"changed_since,omitempty"`
	WhereColumn     string   `json:"where_column,omitempty"`
	WhereOperator   string   `json:"where_operator,omitempty"`
	WhereValues     []string `json:"where_values,omitempty"`
}
```

For the 4 variables in the struct you mentioned, if they are nil or undefined when encoding/decoding the JSON they will be omitted or left out post translation.